src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -145,7 +145,11 @@ public Task PutsMetrics()
                     },
                     [nameof(Database.Metrics.Attachments)] = new DynamicJsonValue
                     {
-                        [nameof(Database.Metrics.Attachments.PutsPerSec)] = Database.Metrics.Docs.PutsPerSec.CreateMeterData(true, empty)
+                        [nameof(Database.Metrics.Attachments.PutsPerSec)] = Database.Metrics.Attachments.PutsPerSec.CreateMeterData(true, empty)
```

Comment:
Same fix needs to be applied to `/databases/*/metrics/bytes`

Reply:
done